ocaml/xapi/importexport.ml

```diff
@@ -277,7 +277,12 @@ let remote_metadata_export_import ~__context ~rpc ~session_id ~remote_address ~r
                         )
                      )
                 )
-             );
+             )
+             with e ->
```

Comment:
In what circumstances can `Xmlrpc_client.Stunnel_connection_failed` happen?
We get it if the destination network is unreachable, but what if we can reach the network but there is just nothing responding on the relevant TCP port of the destination host? What if a TCP connection is made but then TLS negotiation fails?
I think "network is unreachable" is a more specific assumption than we can justify.